src/hardware/pounder/dds_output.rs

```diff
@@ -52,9 +52,11 @@
 ///! compile-time-known register update sequence needed for the application, the serialization
 ///! process can be done once and then register values can be written into a pre-computed serialized
 ///! buffer to avoid the software overhead of much of the serialization process.
+use log::warn;
+use stm32h7xx_hal as hal;
```

Comment:
Any reason why this file was changed? Was it clippy-related?

Reply:
Yes. Because of the removed old style `extern crate log` and knock-on clippy complaints.